appbuilder/project/project-base.ts

```diff
@@ -80,6 +85,56 @@ export abstract class ProjectBase implements Project.IProjectBase {
 		};
 	}
 
+	public getAppIdentifierForPlatform(platform?: string): IFuture<string> {
+		return ((): string => {
+			if (!this._platformSpecificAppIdentifier) {
+				if (platform &&
+					platform.toLowerCase() === this.$projectConstants.ANDROID_PLATFORM_NAME.toLowerCase() &&
+					this.projectData.Framework === TARGET_FRAMEWORK_IDENTIFIERS.Cordova) {
+					let pathToAndroidResources = path.join(this.projectDir, this.$staticConfig.APP_RESOURCES_DIR_NAME, this.$projectConstants.ANDROID_PLATFORM_NAME);
+
+					let pathToAndroidManifest = path.join(pathToAndroidResources, ProjectBase.ANDROID_MANIFEST_NAME);
+					let appIdentifierInAndroidManifest = this.getAppIdentifierFromConfigFile(pathToAndroidManifest, /package\s*="([\S]*)"/).wait();
```

Comment:
You've handled the case where there are whitespace characters **before** the `=` symbol but not **after**